integration-tests/ahm/src/tests.rs

```diff
@@ -303,8 +305,8 @@ async fn migration_works() {
 
 	let mut rc_block_count = 0;
 	// finish the loop when the migration is done.
-	while rc.execute_with(|| RcMigrationStageStorage::<Polkadot>::get()) !=
-		RcMigrationStage::MigrationDone
+	while rc.execute_with(|| RcMigrationStageStorage::<Polkadot>::get())
+		!= RcMigrationStage::MigrationDone
```

Comment:
we gonna move this formatting back and forth

Reply:
this is different between nightly (before) and non-nightly (now) fmt. CI commands use nightly